specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/BulkActions.json

```diff
@@ -0,0 +1,652 @@
+            "description": "Connective used to logically join the values on this clause",
+            "$ref": "#/definitions/Connective"
+          },
+          "field": {
+            "description": "The name of the field we are targeting.",
+            "type": "string"
+          },
+          "operator": {
+            "description": "Operator which describes how this clause should be constructed into an expression.",
+            "type": "string",
+            "enum": [
+              "Equal",
+              "NotEqual",
+              "LessThan",
+              "GreaterThan",
+              "StringContains",
+              "StringNotContains",
+              "IsNull",
+              "IsNotNull"
+            ],
+            "x-ms-enum": {
+              "modelAsString": true,
+              "name": "ConditionOperator",
+              "values": [
+                {
+                  "description": "The '=' operator.",
+                  "value": "Equal"
+                },
+                {
+                  "description": "The '!=' operator.",
+                  "value": "NotEqual"
+                },
+                {
+                  "description": "The '<' operator.",
+                  "value": "LessThan"
+                },
+                {
+                  "description": "The '>' operator.",
+                  "value": "GreaterThan"
+                }, 
+                {
+                  "description": "The 'contains' operator for strings.",
+                  "value": "StringContains"
+                },       
+                {
+                  "description": "The 'not contains' operator for strings.",
+                  "value": "StringNotContains"
+                },              
+                {
+                  "description": "The 'is_null' operator.",
+                  "value": "IsNull"
+                },
+                {
+                  "description": "The 'not is_null' operator.",
+                  "value": "IsNotNull"
+                }
+              ]
+            }
+          },
+          "values": {
+            "description": "Represents all values to construct this clause from in combination with the clause's field and operator.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "x-ms-identifiers": []
+          }
+        }
+      },
+      "Condition": {
+        "description": "Represents a condition which is composed of a set of clauses.",
+        "type": "object",
+        "properties": {
+          "conditionConnective": {
+            "description": "Connective used to logically join the clauses.",
+            "$ref": "#/definitions/Connective"
+          },
+          "clauses": {
+            "description": "List of clauses which make up this condition.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Clause"
+            },
+            "x-ms-identifiers": []
```

Comment:
>x-ms-identifiers": [] [](http://example.com/codeflow?start=13&length=21)

If there are properties that uniquely identify an element in the array, specify that here. You can specify more than one property that uniquely identifies an element.